File: src/main/scala/org/openapitools/server/model/GetBusinessSettingsResponse.scala
package org.openapitools.server.model


/**
 * Ответ на запрос настроек кабинета.
 *
 * @param status  for example: ''null''
 * @param result  for example: ''null''
*/
final case class GetBusinessSettingsResponse (
  status: Option[ApiResponseStatusType] = None,
  result: Option[GetBusinessSettingsInfoDTO] = None
)

File: ympa_fsharp_giraffe_server/src/model/GetBusinessSettingsResponse.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.ApiResponseStatusType
open ympa_fsharp_giraffe_server.Model.GetBusinessSettingsInfoDTO

module GetBusinessSettingsResponse =

  //#region GetBusinessSettingsResponse


  type GetBusinessSettingsResponse = {
    Status : ApiResponseStatusType;
    Result : GetBusinessSettingsInfoDTO;
  }
  //#endregion
